R/names.R

```diff
@@ -158,7 +158,7 @@
 vec_as_names <- function(names,
                          ...,
                          repair = c("minimal", "unique", "universal", "check_unique"),
-                         repair_arg = "",
+                         repair_arg = caller_arg(repair),
```

Comment:
I'm not sure this is right, I feel like it was correct as `""` because we don't want to reference `"minimal"` and friends in the output, we want to say what the name of `repair` was in the top level function we call `vec_as_names()` from

Reply:
I'm not sure I understand what you mean regarding references "minimal". This is looking for the name of `repair` in the caller.

Reply:
I think we need to take `repair_arg = NULL`, and then treat two different cases:

- If `NULL`, the repair strategy is fixed. Wrong values are _internal_ errors that need to use the internal call. We also don't provide recommendations about changing the repair strategy.

- If a string, the repair strategy is provided by the user. Wrong values are _user_ errors and we do want to provide recommendations about repair strategy.

Reply:
I think that sounds right!